app/routes.py

```diff
@@ -9,7 +9,7 @@
 @main_bp.route('/index', methods=['GET'])
 @login_required
 def index():
-    masterclasses = Masterclass.query.filter_by(draft=None).order_by(Masterclass.timestamp.asc()).all()
+    masterclasses = Masterclass.query.filter_by(draft=False).order_by(Masterclass.timestamp.asc()).all()
```

Comment:
I think it makes more sense for `draft` to be set as False when not true, rather than None.

Reply:
Basically before I wiped my local db all of the existing masterclasses just had `None` for the draft field. However after wiping them, I can add them as `False`, as I will when a user comes to the end of the create masterclass flow.